From: Richard Cochran <richardcochran@gmail.com>
Subject: net: export time stamp utility function for Ethernet MAC drivers
Git-commit: 09fac8536d62e65fca8fa126fd3012aae9524c2c
Patch-mainline: no
References: fate#314848
Acked-by: Jiri Bohac <jbohac@suse.cz>

The network stack provides the function, skb_clone_tx_timestamp().
Ethernet MAC drivers can call this via the transmit time stamping
hook, skb_tx_timestamp(). This commit exports the clone function so
that drivers using it can be compiled as modules.

Signed-off-by: Richard Cochran <richard.cochran@omicron.at>
Signed-off-by: David S. Miller <davem@conan.davemloft.net>

diff --git a/net/core/timestamping.c b/net/core/timestamping.c
index c799b0a..82fb288 100644
--- a/net/core/timestamping.c
+++ b/net/core/timestamping.c
@@ -72,6 +72,7 @@ void skb_clone_tx_timestamp(struct sk_buff *skb)
 		break;
 	}
 }
+EXPORT_SYMBOL_GPL(skb_clone_tx_timestamp);
 
 void skb_complete_tx_timestamp(struct sk_buff *skb,
 			       struct skb_shared_hwtstamps *hwtstamps)
-- 
1.7.11.4

